### ISSUE TITLE
Regarding Training epochs 

### ISSUE THREAD

**Issue by punitive1729** [Neerajj9/CSRNet-keras#30]
How many epochs was this model run for..
Also there's no concept of EarlyStopping or monitoring loss
